plugin_xiplot_filetypes/README.md

```diff
@@ -0,0 +1,5 @@
+# [&chi;iplot](https://github.com/edahelsinki/xiplot) plugin for additional file types
+
+This plugin adds support for additional file types (beside `csv` and `json`) to [&chi;iplot](https://github.com/edahelsinki/xiplot).
+Currently, this plugin adds support for `feather` and `parquet`.
+Note that in the [WASM version](https://edahelsinki.fi/xiplot) only support for `parquet` is added.
```

Comment:
Perhaps this could also have a brief explainer for how to install the plugin in WASM?